### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by JRI98**
If there is something that should be kept, please let me know 🙂

### CODE REVIEW

File: crates/compiler/build/src/target.rs [hunk 1]
```diff
@@ -25,22 +25,6 @@ pub fn target_triple_str(target: Target) -> &'static str {
     }
 }
 
-pub fn target_zig_str(target: Target) -> &'static str {
```
Comment (lukewilliamboswell) [inline crates/compiler/build/src/target.rs:28]: This is now no longer required with the removal of host rebuilding I believe. 👍

**Comment by lukewilliamboswell** [comment]
I've looked through these changes and don't see anything out of place. 

I think another review would be sensible; I'm not familiar with some parts being affected here. 

@JRI98 what is your method for finding these unused things? do you have an analysis tool like clippy or similar that you are using?

**Review by ayazhafiz — DISMISSED** [review]

**Comment by lukewilliamboswell** [comment]
@JRI98 are you able to resolve the merge conflict?

**Review by smores56 — APPROVED** [review]

**Comment by JRI98** [comment]
Thank you for pushing this forward. Could only get back to this now.

**Comment by smores56** [comment]
Happy to help

**Pull request merged**